pkg/oc/cli/admin/release/git.go

```diff
@@ -116,16 +117,18 @@ func sourceLocationAsRelativePath(dir, location string) (string, error) {
 	return basePath, nil
 }
 
-type MergeCommit struct {
-	CommitDate time.Time
+type commit struct {
+	// Merkle graph information
+	Hash    string
+	Parents []string
 
-	Commit        string
-	ParentCommits []string
+	// Local Merkle-graph node information
+	Subject       string
+	CommitterDate time.Time
```

Comment:
CommitDate vs CommiterDate ?

Reply:
> CommitDate vs CommiterDate ?

I'm fine leaving that off, but as long as I was touching this struct I thought I'd align with Git's usage (which is not perfectly consistent either):

```sh
$ git clone git://github.com/git/git.git
$ cd git
$ git grep -i committer.date v2.21.0 | wc -l
330
$ git grep -i commit.date v2.21.0 | wc -l
79
$ git grep -i committerdate v2.21.0 | wc -l
14
$ git grep -i commitdate v2.21.0 | wc -l
2
```

That said, `CommitDate` is the default in pretty logs (although not always a winner in the pretty docs :p):

```console
$ git grep -i 'commit.*date' Documentation/pretty-formats.txt | cat
Documentation/pretty-formats.txt:	  CommitDate: <committer date>
Documentation/pretty-formats.txt:- '%cd': committer date (format respects --date= option)
Documentation/pretty-formats.txt:- '%cD': committer date, RFC2822 style
Documentation/pretty-formats.txt:- '%cr': committer date, relative
Documentation/pretty-formats.txt:- '%ct': committer date, UNIX timestamp
Documentation/pretty-formats.txt:- '%ci': committer date, ISO 8601-like format
Documentation/pretty-formats.txt:- '%cI': committer date, strict ISO 8601 format
```

So I'm leaning weakly towards `CommitterDate` but could be swayed ;).